src/python/review/inspectors/flake8/issue_types.py

```diff
@@ -15,12 +15,90 @@
 
     # builtin naming
     'A003': IssueType.BEST_PRACTICES,
+
+    # flake8-broken-line
+    'N400': IssueType.CODE_STYLE,
+
+    # flake8-commas
+    "C812": IssueType.CODE_STYLE,
```

Comment:
This string is in double quotes `"`, while the previous one is in single quotes `'`. Please be more consistent with quote types.